<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sender</class>
 <widget class="QDialog" name="sender">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>538</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>卡萨帝 wifi 控制台</string>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="sendButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send to...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>33</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>选择网卡</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="sendTextEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>设备地址</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="hex_checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clearButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>75</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>:8266</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="auto_enter_checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto \n</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>471</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>511</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_find_client">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始查找设备</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_device_on_line">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>471</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fan_set_temp">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fan_current_temp">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>360</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fan_speed">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fan_cel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>440</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_fan_on_off_state">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>480</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_solder_current_temp">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_solder_on_off_state">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>480</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_solder_set_temp">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_empty">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>400</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_solder_cal">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>440</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>320</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>风机设定温度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>360</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>风机当前温度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>404</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>风扇转速</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>440</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>风机校准值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>480</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>风机开机状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>273</x>
     <y>480</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>焊台开机状态</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>273</x>
     <y>320</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>焊台设定温度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>440</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>焊台校准值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>273</x>
     <y>360</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>焊台当前温度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>403</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>空...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_read_fan_set_temp">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_write_fan_set_temp">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_write_fan_current_temp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_read_fan_current_temp">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_read_fan_cal">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>440</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_read_fan_speed">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>400</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_write_fan_speed">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_write_fan_cal">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>440</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_read_fan_on_off_state">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>480</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_write_fan_on_off_state">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>480</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_write_solder_cal">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>440</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_write_solder_on_off_state">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>480</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_read_solder_set_temp">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_empty_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>400</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_read_solder_current_temp">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>360</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_write_solder_set_temp">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>320</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_write_solder_current_temp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>360</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_read_solder_on_off_state">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>480</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_read_solder_cal">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>440</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_empty">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>400</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <zorder>pushButton_read_solder_cal</zorder>
  <zorder>sendButton</zorder>
  <zorder>comboBox</zorder>
  <zorder>label</zorder>
  <zorder>lineEdit</zorder>
  <zorder>sendTextEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>hex_checkBox</zorder>
  <zorder>textBrowser</zorder>
  <zorder>clearButton</zorder>
  <zorder>label_4</zorder>
  <zorder>auto_enter_checkBox</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>pushButton_find_client</zorder>
  <zorder>label_device_on_line</zorder>
  <zorder>line_3</zorder>
  <zorder>lineEdit_fan_set_temp</zorder>
  <zorder>lineEdit_fan_current_temp</zorder>
  <zorder>lineEdit_fan_speed</zorder>
  <zorder>lineEdit_fan_cel</zorder>
  <zorder>lineEdit_fan_on_off_state</zorder>
  <zorder>lineEdit_solder_current_temp</zorder>
  <zorder>lineEdit_solder_on_off_state</zorder>
  <zorder>lineEdit_solder_set_temp</zorder>
  <zorder>lineEdit_empty</zorder>
  <zorder>lineEdit_solder_cal</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>label_15</zorder>
  <zorder>label_16</zorder>
  <zorder>pushButton_read_fan_set_temp</zorder>
  <zorder>pushButton_write_fan_set_temp</zorder>
  <zorder>pushButton_write_fan_current_temp</zorder>
  <zorder>pushButton_read_fan_current_temp</zorder>
  <zorder>pushButton_read_fan_cal</zorder>
  <zorder>pushButton_read_fan_speed</zorder>
  <zorder>pushButton_write_fan_speed</zorder>
  <zorder>pushButton_write_fan_cal</zorder>
  <zorder>pushButton_read_fan_on_off_state</zorder>
  <zorder>pushButton_write_fan_on_off_state</zorder>
  <zorder>pushButton_write_solder_cal</zorder>
  <zorder>pushButton_write_solder_on_off_state</zorder>
  <zorder>pushButton_read_solder_set_temp</zorder>
  <zorder>pushButton_empty_2</zorder>
  <zorder>pushButton_read_solder_current_temp</zorder>
  <zorder>pushButton_write_solder_set_temp</zorder>
  <zorder>pushButton_write_solder_current_temp</zorder>
  <zorder>pushButton_read_solder_on_off_state</zorder>
  <zorder>pushButton_empty</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
